usr.sbin/bsdinstall/scripts/zfsboot

```diff
@@ -906,7 +906,7 @@ zfs_create_diskpart()
 			fi
 		fi
 
-		# NB: zpool will use the `zfs#' GPT labels
+		# NB: ZFS pool will use `zfs#' GPT labels
```

Comment:
"ZFS pools" to agree with "labels" object

Reply:
Fixed by [`8ff856d` (#1569)](https://github.com/freebsd/freebsd-src/pull/1569/commits/8ff856db200c88b03bca469410cd36ff9e602ee7). 

Thanks @bsdjhb for the catch.